packages/core/src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-processor.ts

```diff
@@ -47,15 +64,16 @@ export default class AmazonPayV2PaymentProcessor {
         checkoutState: InternalCheckoutSelectors,
         methodId: string,
         placement: AmazonPayV2Placement,
-        options?: AmazonPayV2ButtonParameters
+        options?: AmazonPayV2ButtonParameters,
+        decoupleCheckoutInitiation = false
```

Comment:
Can we combine the options and `decoupleCheckoutInitiation`. Or move the `decoupleCheckoutInitiation` before since optional params should be in the end

Reply:
Instead, I converted the parameters to an object.